drivers/hmis/app/models/hmis/hud/household.rb

```diff
@@ -113,7 +113,9 @@ def self.sort_by_option(option)
         hh_t[:any_wip].eq(true).desc,
         hh_t[:latest_exit].eq(nil).desc,
         earliest_entry: :desc,
+        id: :desc,
       )
+    # FIXME these HoH sort orders aren't working as desired, they sort based on all household members
```

Comment:
@martha I found a bug while reviewing these, it looks like the sort is actually based on all household members, not just the HoH. I think it's low prio to fix, but would you mind adding a ticket to the backlog? thank you!

<img width="1403" alt="Screenshot 2024-05-01 at 8 53 40 AM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/6b826047-e67a-4ad3-8b05-f0141c236701">

Reply:
Yes, sorry about that! Created https://github.com/open-path/Green-River/issues/6002

Reply:
thanks!